codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/customizations/ResiliencyConfigCustomization.kt

```diff
@@ -430,12 +430,12 @@ class ResiliencyReExportCustomization(private val runtimeConfig: RuntimeConfig)
         }
         rustCrate.withModule(ClientRustModule.Config.retry) {
             rustTemplate(
-                "pub use #{types_retry}::{RetryConfig, RetryConfigBuilder, RetryMode};",
+                """
+                pub use #{types_retry}::{RetryConfig, RetryConfigBuilder, RetryMode, ReconnectMode};
+                pub use #{runtime_types_retry}::RetryPartition;
```

Comment:
This should only be re-exported in orchestrator mode.

Also, it looks like aws-smithy-runtime snuck into middleware mode... so if you want to track that down, that would be fantastic 😆

Reply:
Good catch. I found a few of them.